spec/lib/environment_handler_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'spec_helper'
+include EnvironmentHandler
+
+describe EnvironmentHandler do
+  describe '.env?' do
```

Comment:
As stated above, I think that we should include an empty string as being "falsey". We should also add a test for that.